src/main/java/edu/hm/hafner/analysis/parser/checkstyle/TopicRule.java

```diff
@@ -56,7 +58,9 @@ public void end(final String namespace, final String name) throws Exception {
     @SuppressFBWarnings("SECURITY")
     private String extractNodeContent(final Element subsection) throws TransformerException {
         StringWriter content = new StringWriter();
-        Transformer transformer = TransformerFactory.newInstance().newTransformer();
+        TransformerFactory transformerFactory = TransformerFactory.newInstance();
+        secureFactory(transformerFactory);
+        Transformer transformer = transformerFactory.newTransformer();
```

Comment:
Can you please move that code (and the private methods) to the `SecureXmlParserFactory` class as well? Then all of those security related factory methods are in one place?

Reply:
I did but I had to create a public static method for that.

Reply:
What I mean is that you can use it in the same way:
```
Transformer transformer = new SecureXmlParserFactory().createTransformer();